code/datums/flock/flock.dm

```diff
@@ -489,22 +447,35 @@
 /datum/flock/proc/reserveTurf(var/turf/simulated/T, var/name)
 	if(T in all_owned_tiles)
 		return
-	if(T in src.busy_tiles)
-		return //can't reserve tiles that are already reserved
+	if(name in src.busy_tiles)
+		return
 	src.busy_tiles[name] = T
-	src.updateAnnotations()
+
+	var/image/I = image('icons/misc/featherzone.dmi', T, "frontier")
+	I.blend_mode = BLEND_ADD
+	I.alpha = 80
+	I.plane = PLANE_ABOVE_LIGHTING
+	src.annotations_busy_tiles[T] = I
+	src.addClientImage(I)
 
 /datum/flock/proc/unreserveTurf(var/name)
+	var/turf/simulated/T = src.busy_tiles[name]
 	src.busy_tiles -= name
-	src.updateAnnotations()
+
+	var/image/I = src.annotations_busy_tiles[T]
```

Comment:
Should probably make sure `istype(T)` here, just in case some bad call to `unreserveTurf(bad name)` or something fucky happens with turf deletion.

Reply:
I think ideally this would be handled by onUpdate() of /datum/action/bar/flock_convert?